docs/02-app/01-building-your-application/01-routing/06-redirecting.mdx

```diff
@@ -252,6 +252,86 @@ export const config = {
 
 See the [Middleware](/docs/app/building-your-application/routing/middleware) documentation for more information.
 
+## External redirects
+
+App Router fully supports programmatic external redirects to other sites (ie. `https://google.com`) using the same methods mentioned above.
+
+An example of when this would be useful is when generating checkout urls for hosted payment providers, such as Stripe and Shopify.
+
+[`redirect`](#redirect-function) and [permanentRedirect`](#permanentredirect-function) work on the server and in [Server Actions](/docs/app/building-your-application/data-fetching/server-actions-and-mutations).
+
+```ts filename="app/actions.ts" switcher
+'use server'
+
+import { redirect } from 'next/navigation'
+import { createCheckout } from './payment'
+
+export async function checkout(data: FormData) {
+  // Do validation and / or generate url
+  const { url } = await createCheckout(data)
+
```

Comment:
I wonder if we even need this, or we just keep the example super brief with the `redirect('https://google.com')` or similar.